### DISCUSSION TITLE
How to limit the number of listing items?

### DISCUSSION THREAD

**Discussion by WillemSleegers** [quarto-dev/quarto-cli#622] [Q&A] [answered]
Is it possible to limit the number of items that are listed in a listing, without using pagination? I want to list only the 3 or 4 most recent blog posts on my Home page.

I could enable pagination, but this doesn't look as nicely in the middle of a page and clicking on the pages also doesn't automatically move the user to the top of the list of blog posts.

**Comment by jjallaire** [comment]
We don't have an option to to do this now but we will add one! (transferring this to a GH issue)

**Reply by WillemSleegers** [reply]
Thanks!

**Comment by dragonstyle** [comment, marked as answer]
We do support this now via `max-items`: https://quarto.org/docs/websites/website-listings.html#listing-options

(just noticed that there isn't a linked issue so this loop never closed). Thanks for pointing this out it is a great addition!

**Reply by WillemSleegers** [reply]
That sounds great, thanks!
